integration-tests/api/test/api-entries.test.js

```diff
@@ -45,4 +45,14 @@ describe('@opentelemetry/api entries', () => {
     const mod = require('@opentelemetry/api/experimental');
     assert.ok(mod.wrapTracer != null);
   });
+
+  it('should import baggage utils', async () => {
+    const mod = await import('@opentelemetry/core');
+    assert.ok(mod.baggageUtils.getKeyPairs != null);
+  });
+
+  it('should require baggage utils', () => {
+    const mod = require('@opentelemetry/core');
+    assert.ok(mod.baggageUtils.getKeyPairs != null);
+  });
```

Comment:
I think this should go into a separate `describe` block, maybe a separate file even, since this is intended to test the API package.

Reply:
[Moved to a separate file](https://github.com/open-telemetry/opentelemetry-js/pull/4880/commits/f0124ea6697f964d6714136a4b67bdea6db0cdd5) (but still within the `./integration-tests/api` directory 🤷🏻) _and_ updated the `describe` and `it` statements to make more explicit what is being asserted.